crawl-ref/source/mon-util.cc

```diff
@@ -2627,6 +2627,11 @@ mon_spellbook_type get_spellbook(const monster_info &mon)
 vector<mon_spell_slot> get_unique_spells(const monster_info &mi,
                                mon_spell_slot_flags flags)
 {
+    vector<mon_spell_slot> slots;
+    // the rest of this function crashes if mi has no spells
```

Comment:
Minor: we can cut the comment. It's fairly intuitive, and very easy to discover if someone tries to delete it :)